ini-templates/development-template.ini

```diff
@@ -50,8 +50,8 @@ set queue_worker_get_size = 2000
 [composite:visindexer]
 use = config:base.ini#visindexer
 
-[composite:regionindexer]
-use = config:base.ini#regionindexer
+#[composite:regionindexer]
+#use = config:base.ini#regionindexer
```

Comment:
Should be removed in separate ticket?

Reply:
Can be done, but region search indexing tests on the main branch won't pass. Then, this would need a closer scrutiny when merging the branch with a single failing test.